guides/plugins/plugins/administration/using-assets.md

```diff
@@ -0,0 +1,70 @@
+# Using assets
+
+## Overview
+
+When working with an own plugin, the usage of own custom images or other assets is a natural requirement. 
+So of course you can do that in Shopware as well. In this guide we will explore how you can
+add and use custom assets in your plugin in order to use them in the administration.
+
+## Prerequisites
+
+In order to be able to start with this guide, you need to have an own plugin running. As to most guides, this guide
+is also built upon the Plugin base guide:
+
+{% page-ref page="../plugin-base-guide.md" %}
```

Comment:
Why do we use `page-ref` here ? I guess this displayes as a nice gitbook card ?
Most guides use plain markdown links: `()[]` in prerequisites.

Reply:
Yes, I wanted to use this shiny gitbook features but as it was broken before, it was waived in the articles before (or you got broken articles like me 🙈 )